src/nxt_http_route.c

```diff
@@ -1594,8 +1626,15 @@ nxt_http_route_match(nxt_task_t *task, nxt_http_request_t *r,
     nxt_http_route_match_t *match)
 {
     nxt_int_t              ret;
+    nxt_http_action_t      *action;
     nxt_http_route_test_t  *test, *end;
 
+    ret = nxt_http_cond_value(task, r, &match->condition);
+    if (ret <= 0) {
+        action = (nxt_http_action_t *) (intptr_t) ret;
+        return action;
```

Comment:
Minor thing, feel free to ignore...

Do we need the `action =` step rather than just returning the thing directly (casted of course).

Reply:
Makes sense.